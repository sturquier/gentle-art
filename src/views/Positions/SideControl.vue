<template>
  <h1>Side control</h1>
</template>
