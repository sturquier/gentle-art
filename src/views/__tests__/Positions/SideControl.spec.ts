import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import SideControl from '../../Positions/SideControl.vue'

describe('SideControl', () => {
  it('renders properly', () => {
    const wrapper = mount(SideControl)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
