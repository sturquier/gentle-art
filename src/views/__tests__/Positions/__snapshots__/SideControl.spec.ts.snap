// Vitest Snapshot v1

exports[`SideControl > renders properly 1`] = `"<h1>Side control</h1>"`;
